string[] joints_names
float32[] positions
---
bool success
string message